src/natcap/invest/validation.py

```diff
@@ -499,18 +515,24 @@ def check_boolean(value):
             type(value), value)
 
 
-def check_csv(filepath, required_fields=None, excel_ok=False):
+def check_csv(filepath, rows=None, columns=None, excel_ok=False, **kwargs):
     """Validate a table.
 
     Args:
         filepath (string): The string filepath to the table.
-        required_fields=None (list): A case-insensitive list of fieldnames that
-            must exist in the table.  If None, fieldnames will not be checked.
+        rows (dict): A list of regex-compilable patterns that
```

Comment:
Also double checking this regex commentary should still be here?

Reply:
fixed in dc72cfc1